src/utils.js

```diff
@@ -1,3 +1,5 @@
+import { isEqualWith } from 'lodash';
```

Comment:
Can make internal function if needed

Reply:
The `values` object is not deep or complex,  it is a shallow state array _typically_,  I'd recommend you simply iterate it typically comparing by key.
If the user uses a complex `value` (aka, an `object`), then it should be treated naively.  We won't be able to predict what is sane there.